compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp

```diff
@@ -1194,6 +1194,61 @@ void propagate_one_hot_quantparam(luci::CircleOneHot *one_hot, loco::DataType qu
   quant_input(&CircleOneHot::off_value, &CircleOneHot::off_value);
 }
 
+void quantize_one_hot_indices(luci::CircleOneHot *one_hot, loco::DataType quant_type)
+{
+  auto *indices = loco::must_cast<luci::CircleNode *>(one_hot->indices());
+
+  // skip if indices has not only OneHot successors
+  {
+    auto const succs = loco::succs(indices);
+    auto is_one_hot = [](auto *node) { return dynamic_cast<CircleOneHot *>(node) != nullptr; };
+    if (not std::all_of(begin(succs), end(succs), is_one_hot))
+      return;
+  }
+
+  // define depth of OneHot
+  auto const *depth_node = dynamic_cast<luci::CircleConst *>(one_hot->depth());
+  if (depth_node == nullptr)
+    throw std::runtime_error("Non const OneHot's depth is unsupported.");
```

Comment:
Q) Is `depth_node` being nullpt wrong network or can be but we cannot support this?
As before, if this can happen we should just skip if not throw is OK.